types/src/v2/error.rs

```diff
@@ -64,7 +65,7 @@ pub struct ErrorObject<'a> {
 	/// Code
 	pub code: ErrorCode,
 	/// Message
-	pub message: &'a str,
+	pub message: Cow<'a, str>,
```

Comment:
```suggestion
	#[serde(borrow)]
	pub message: Cow<'a, str>,
```

Otherwise it always deserializes to owned variant.

Reply:
danke :pray: